docx-asciidoc-conversion/README.md

```diff
@@ -16,39 +17,42 @@ The procedure was tested using the INSPIRE Metadata Technical Guidance 2.0. The
 
 ### Conversion Procedure
 
-1. Correct the original docx document, save as `md_2_fixed.docx`:
-   - apply *XML Example* style to all XML examples (about 7 cases were found in `md_2_0_1.docx` where this fix was required)
-   - remove paragraph breaks from inside footnotes (two cases were found in `md_2_0_1.docx` where this fix was required)
+1. Create a new folder with the name of the document to convert:
+   ```shell
+    mkdir my_example_data_specifications_EX
+    ```
 
-2. Convert the corrected docx document to *AsciiDoc* using Pandoc, save as `md_2_pandoc.adoc`, extract images to `media` folder:
+2. Copy the Microsoft Word document into the new folder. Convert the Microsoft Word document into the last .docx format:
```

Comment:
could you clarify what 'last .docx format' means?

Reply:
Description improved